### DISCUSSION TITLE
how to input expected returns vector create by myself

### DISCUSSION THREAD

**Discussion by MaNannn** [dcajasn/Riskfolio-Lib#214] [Q&A]
Hi,
It's a really useful lib, but i got a problem. I was wondering how to input my own  expected returns of each assets instead of using the assets_stats() function to estimates. Thanks a lot!

**Comment by dcajasn** [comment]
Hi @MaNannn, there is an example with custom estimates in the examples section of the documentation.

Best,
Dany
